### ISSUE TITLE
DeclarationOrder false positives in 0.36.0

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#415]
Running scss-lint on the following file:

```scss
.my-selector {
  @include my-mixin;
  border-radius: 0;
  padding: 0;
  position: relative;
}
```

fails in 0.36.0 with:

> test.scss:4 [W] DeclarationOrder: Expected item on line 4 to appear before line 3. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

If I remove any one of these lines, the message goes away.

I've noticed a number of other places in our code where this happens now, but I'm not sure if they are from the same root cause or not.

**Comment by sds** [comment]
Fixed in aa0f8d06a1480c1f20834b025b5e3181ac06d64f.
